backstop/src/backstop/user.rs

```diff
@@ -545,44 +474,4 @@ mod tests {
         // verify exp is respected when specified
         user.dequeue_shares_for_withdrawal(&e, to_dequeue, true);
     }
-
-    #[test]
-    #[should_panic(expected = "Error(Contract, #2)")]
-    fn test_try_withdraw_shares_over_total() {
```

Comment:
this test is still required IMO. It test's the final code block in `dequeue_shares_for_withdrawal`:
```
        if left_to_dequeue > 0 {
            panic_with_error!(e, BackstopError::InvalidBalance);
        )
```